test/check-application

```diff
@@ -371,7 +371,11 @@ class TestApplication(testlib.MachineCase):
 
         self.performPodAction("pod-1", "system", "Delete")
         b.click(".pf-v5-c-modal-box button:contains(Delete)")
-        b.wait_in_text(".pf-v5-c-modal-box__body", "running or paused containers cannot be removed without force")
+        # https://github.com/containers/podman/issues/19159
+        if podman_version(self) >= (4, 6, 0):
```

Comment:
So if the podman issues is fixed, this will change again?  What about just waiting for any kind of error to appear?  (Likely a .pf-v5-c-alert thing, no?)

Reply:
Oh, actually might be a good idea.